### DISCUSSION TITLE
给合作伙伴在蓝盾手机app，构建流水线、下载构建结果的权限

### DISCUSSION THREAD

**Discussion by gladhu** [Tencent/bk-ci#4910] [General]
给合作伙伴在蓝盾手机app，构建流水线、下载构建结果的权限

**Comment by iccengan** [comment]
因安全问题，蓝盾App暂时没有规划 外部人员查看/构建流水线的功能。
如果想让外部人员下载构建产物，可以通过版本体验插件/版本体验服务来满足。具体指引欢迎联系DevOps-helper哈
